packages/core/src/dialects/ibmi/query-generator-typescript.ts

```diff
@@ -61,6 +62,21 @@ export class IBMiQueryGeneratorTypeScript extends AbstractQueryGenerator {
     ]);
   }
 
+  renameTableQuery(
+    beforeTableName: TableNameOrModel,
+    afterTableName: TableNameOrModel,
+    _options?: RenameTableQueryOptions,
```

Comment:
We should add a rejectInvalidOptions with RENAME_TABLE_QUERY_SUPPORTABLE_OPTIONS so it's consistent with other methods